modules/api_management/main.tf

```diff
@@ -136,6 +136,20 @@ resource "azurerm_api_management_diagnostic" "apim_diagnostic" {
   }
 }
 
+resource "azurerm_monitor_smart_detector_alert_rule" "example" {
+  count = length(var.alert_rules_settings)
```

Comment:
Missing fmt formatter ;) all the = signs are alligned if you use the formatter

Reply:
Ah, :) my bad